### ISSUE TITLE
Detect wrong language (CSS instead C#)

### ISSUE THREAD

**Issue by ngohungphuc** [github/linguist#4353]
## Preliminary Steps

Please confirm you have...
- [x] reviewed [How Linguist Works](https://github.com/github/linguist#how-linguist-works),
- [x] reviewed the [Troubleshooting](https://github.com/github/linguist#troubleshooting) docs,
- [x] considered implementing an [override](https://github.com/github/linguist#overrides),
- [x] verified an issue has not already been logged for your issue ([linguist issues](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+repo%3Agithub/linguist)).

## Problem Description
Hi  team. Currently my project is detect wrong language.
Basically my github repo is C# project but in the graph show css and when I click on css link
https://github.com/Awesome-CMS-Core/Awesome-CMS-Core/search?l=css
I cant find any css file ? 
Any idea to fix this.
Best regards.

![chrome_tegckignyd](https://user-images.githubusercontent.com/13591213/49986641-3b090880-ffa3-11e8-9552-f4223db417c7.png)

### URL of the affected repository:
https://github.com/Awesome-CMS-Core/Awesome-CMS-Core/search?l=css
### Expected language:

C#
### Detected language:

CSS

**Comment by lildude** [comment]
This is a result of the language grouping Linguist does that search doesn't. SCSS is grouped under CSS 
https://github.com/github/linguist/blob/fa493000a594f5fbd457bbd473ce791d95b227cc/lib/linguist/languages.yml#L4262-L4265 and your repo has a lot of [SCSS files](https://github.com/Awesome-CMS-Core/Awesome-CMS-Core/search?l=SCSS) and their total bytes of code exceed everything else in your repo, hence the CSS classification as the most prominent language in your repo.

**Comment by lildude** [comment]
And looking at the override you've implemented, looks like you worked this out for yourself too 😁 (though you seem to have included lines from your shell in the file too). Just be warned that your SCSS syntax highlighting is going to be a bit of a mess as you've marked those files as C# so that grammar will be used to highlight the files.

**Comment by Alhadis** [comment]
> This is a result of the language grouping Linguist does that search doesn't. SCSS is grouped under CSS

[All the more reason to separate it](https://github.com/github/linguist/issues/4291). 😉

**Comment by ngohungphuc** [comment]
@lildude yep I just make small work around

**Comment by Alhadis** [comment]
Good to see you've got things fixed! 😀

Since this issue appears resolved, I'm going to close it. Feel free to speak up if you encounter further difficulties. 👍
